build_tools/kokoro/gcp_ubuntu/bazel/linux/x86/swiftshader/continuous.cfg

```diff
@@ -0,0 +1,19 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
This file isn't necessary. There is no "continuous" job anymore (it's been replaced by the "main" and "google" jobs). Looks like I forgot to clean up the leftover files. Opened #2649

Reply:
Yeah. I'm basically just copy paste. :)